src/bloqade/pyqrack/device.py

```diff
@@ -112,6 +162,51 @@ def pauli_expectation(pauli: list[Pauli], qubits: list[PyQrackQubit]) -> float:
 
         return sim_reg.pauli_expectation(qubit_ids, pauli)
 
+    @staticmethod
+    def reduced_density_matrix_rdm(
```

Comment:
Very much optional, but this now just reads as "reduced-density-matrix-reduced-density-matrix" in my mind, so I'm not sure about the naming 😅